src/service/cli/server.js

```diff
@@ -11,6 +11,7 @@ require(`dotenv`).config();
 
 const getIndexRouter = require(`../routers`);
 
+const {getSequelize} = require(`../lib/sequelize`);
```

Comment:
лучше избегать названий фреймворков, у нас sequelize отвечает за db,  так лучше и назвать

Reply:
@Baristanko я поправил, и пока оставил так как есть, именно для sequelize. Объясню почему.
В документации сказано: 
```Terminology convention
Observe that, in the examples above, Sequelize refers to the library itself while sequelize refers to an instance of Sequelize, which represents a connection to one database. This is the recommended convention and it will be followed throughout the documentation.```
То есть тут это "хорошая практика". 
Если я не прав, то я переделаю в следующем задании, дай знать, если я не прав)

Reply:
мне кажется они и обращают внимание, что такие названия они используют именно для документации)
Вообще все инфраструктурные штуки обычно называют по роли какую они реализуют.
db, logger, config) мы же не называем logger -> pino,  config->dotenv и тд
Поправь тогда в след задании